ci-operator/config/openshift/grafana/openshift-grafana-master.yaml

```diff
@@ -0,0 +1,30 @@
+base_images:
+  base:
+    cluster: https://api.ci.openshift.org
+    name: origin-v4.0
+    namespace: openshift
+    tag: base
+canonical_go_repository: github.com/openshift/grafana
+images:
+- from: base
+  to: grafana
```

Comment:
You need a dockerfile path here?

Reply:
If I don't specify one, it will just default to 'Dockerfile' right?  I think that's ok.

Reply:
Good question -- I think that might be the emergent behavior from the system since we are using OpenShift Builds under the covers? But I am not sure :)

Reply:
We didn't specify 'dockerfile_path' in the other monitoring component configs.  For example, https://github.com/openshift/release/blob/2accb6f042c83ca0497af42c1c0845fd4865887a/ci-operator/config/openshift/prometheus/openshift-prometheus-master.yaml
And that one seems to build building ok.